cincinnati/src/lib.rs

```diff
@@ -799,8 +799,12 @@ pub mod testing {
     use super::*;
     use crate::plugins::internal::versioned_graph::VersionedGraph;
 
-    pub fn generate_graph() -> Graph {
+    pub fn generate_graph(include_conditional_edge: bool) -> Graph {
```

Comment:
We should add a comment here about why are adding this option of generating conditional edges based on `include_conditional_edge: bool`

Reply:
I meant we should communicate that because it is not possible to test the conditional edges in tests we will use generate_graph(false) for the tests and in future we will fix this